### PR TITLE
feat: added Forward Email (https://forwardemail.net), updated notes on Proton Mail, Tutanota, and Skiff

### PR DESCRIPTION
**Pull request by titanism**
### Type

Addition

---

### Changes

We've added Forward Email (https://forwardemail.net) and updated accuracy of notes regarding "open source" claims for other providers.

---

### Supporting Material

See <https://github.com/Lissy93/email-comparison/pull/13> for supporting material and notes (similar PR to another repo owned by @Lissy93) 🙏.

---

### Affiliation

We are the creators and owners of the service.

---

### Checklist

- [x] I have read the [Contributing](https://github.com/Lissy93/awesome-privacy/blob/main/.github/CONTRIBUTING.md) guide, and confirmed my PR aligns with the requirements
- [x] I have performed a self-review (valid Markdown formatting, spelling, and grammar)
- [x] I have indicated whether I have any affiliation with any software / services added  
- [x] I agree to follow the repositories [Contributor Covenant Code of Conduct](https://github.com/Lissy93/awesome-privacy/blob/main/.github/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,11 +276,12 @@ The below email providers are private, end-to-end encrypted (E2EE) and reasonabl
...
+**[Forward Email](https://forwardemail.net)** | 100% open-source and privacy-focused encrypted email service that supports [SMTP](https://forwardemail.net/faq#do-you-support-sending-email-with-smtp), [IMAP](https://forwardemail.net/faq#do-you-support-receiving-email-with-imap), [POP3](https://forwardemail.net/faq#do-you-support-pop3), [email API](https://forwardemail.net/faq#do-you-support-sending-email-with-api), [email webhooks](https://forwardemail.net/en/faq#do-you-support-webhooks), [regex](), unlimited aliases, unlimited domains, [OpenPGP/WKD](https://forwardemail.net/faq#do-you-support-openpgpmime-end-to-end-encryption-e2ee-and-web-key-directory-wkd), and more.  [Paid plans start at $3/mo for an enhanced set of features](https://forwardemail.net/private-business-email?pricing=true).  It is compatible with Apple Mail, Gmail, Mozilla, Thunderbird, K-9 Mail, and any open-source email client.  Unlike other email providers, it is 100% open-source and uses [individually encrypted SQLite mailboxes](https://forwardemail.net/blog/docs/best-quantum-safe-encrypted-email-service) to store your emails, where only you have the password.
```
Comment (Lissy93) [inline README.md:279]: > `[regex]()`

Missing link. (nice feature btw, haven't seen this on other email forwarders!)


> `It is compatible with Apple Mail, Gmail, Mozilla, Thunderbird, K-9 Mail, and any open-source email client.`

Just say what protocols it works with, this is unecissary. 


> `Unlike other email providers`

Reads like an advert. Let's keep things objective.

File: README.md [hunk 2]
```diff
@@ -276,11 +276,12 @@ The below email providers are private, end-to-end encrypted (E2EE) and reasonabl
...
+**[Tutanota](https://tutanota.com/)** | Free and open source ([front-end only](https://old.reddit.com/r/tutanota/comments/10hghin/tutanota_opens_backend_server_side/j5in1wp/)) email service based in Germany. It has a basic intuitive UI, secure native mobile apps, anonymous signup, and a .onion site. Tutonota has a full-featured free plan or a premium subscription for businesses allowing for custom domains ($12/ month).<br>Tutanota [does not use OpenPGP](https://tutanota.com/blog/posts/differences-email-encryption/) like most encrypted mail providers, instead they use a standardized, hybrid method consisting of a symmetrical and an asymmetrical algorithm (with 128 bit AES, and 2048 bit RSA). This causes compatibility issues when communicating with contacts using PGP. But it does allow them to encrypt much more of the header data (body, attachments, subject lines, and sender names etc) which PGP mail providers cannot do
```
Comment (Lissy93) [inline README.md:281]: Changes to other entries should be made as a separate PR. Worried there's a bit of a conflict of interest otherwise.

**Review by lissy93 — CHANGES_REQUESTED** [review]
Unless I've misunderstood your service, shouldn't this be listed under [Mail Forwarding](https://github.com/Lissy93/awesome-privacy#anonymous-mail-forwarding), and not mail providers?

**Comment by Lissy93** [comment]
Also I'm (slowly!) working my way through entries, updating them, but also shortening and them to be more factual and to-the-point. So could you make sure yours doesn't contain anything which reads too much like an ad.

**Review by lissy93 — APPROVED** [review]

**Pull request merged**